wallet/src/de/schildbach/wallet/ui/main/WalletTransactionsFragment.kt

```diff
@@ -156,28 +171,21 @@ class WalletTransactionsFragment : Fragment(R.layout.wallet_transactions_fragmen
         viewModel.isBlockchainSynced.observe(viewLifecycleOwner) { updateSyncState() }
         viewModel.blockchainSyncPercentage.observe(viewLifecycleOwner) { updateSyncState() }
         viewModel.transactions.observe(viewLifecycleOwner) { transactionViews ->
-            val currentLifecycle = viewLifecycleOwner.lifecycle
-            if (viewLifecycleOwner.lifecycle.currentState.isAtLeast(Lifecycle.State.STARTED)) {
-                binding.loading.isVisible = false
```

Comment:
this group of changes was not necesssary. I was trying to prevent the problem seen here:

```kotlin
                    override fun onItemRangeInserted(positionStart: Int, itemCount: Int) {
```

But didn't read the stack trace well enough.